js/app.js

```diff
@@ -1,20 +1,82 @@
 $(document).ready(function() {
-}
+	$('.start').on('click', function(e) {
+		e.preventDefault();
+		$('#intro-container').hide();
```

Comment:
We want to show and hide the whole parent container... all the child containers will follow suit.